devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -30,6 +30,7 @@ metadata {
 
 		// eWeLink
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink SmartPlug (SA-003)", ocfDeviceType: "oic.d.smartplug"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "ZB-SW01", deviceJoinName: "eWeLink 1 Gang Switch"
```

Comment:
Can you add the Jira to the title for this device?

Reply:
I assume that the eWeLink device will be added later?

Reply:
that's my assumption too

Reply:
@FenMei007S It looks like one of your commits had removed this fingerprint, so you will need to create a new PR for it if it is supposed to be added.